### PR TITLE
feat(test): introduce basic e2e tests

### PR DESCRIPTION
**Pull request by steebchen**

### CODE REVIEW

File: examples/simple/simple_test.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+func TestSimpleRun(t *testing.T) {
+	testutils.Prepare(t)
+
+	ch := make(chan interface{}, 1)
+
+	go func() {
+		time.Sleep(5 * time.Second)
+		ch <- struct{}{}
+		log.Printf("sent interrupt")
+	}()
```
Comment (steebchen) [inline examples/simple/simple_test.go]: this just makes it work for now (so that test exits)

File: examples/simple/simple_test.go [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+	if err := run(ch); err != nil {
+		t.Fatalf("run() error = %v", err)
+	}
+
+	// TODO test here that the workflow ran by checking the database
```
Comment (steebchen) [inline examples/simple/simple_test.go]: This is obviously missing and important, but I'm not sure about the best way to do it right now

File: examples/action-old/main.go [hunk 3]
```diff
@@ -0,0 +1,120 @@
+package main
```
Comment (steebchen) [inline examples/action-old/main.go:1]: we might also just delete this package now @abelanger5

Comment (abelanger5) [reply examples/action-old/main.go:1]: I'm updating the `RegisterAction` usage today, so let's just keep it and I'll fix this case.

**Review by abelanger5 — APPROVED** [review]

**Pull request merged**